### PR TITLE
MIXEDARCH-306: Enable upgrade jobs for Azure multi-arch compute in the release controller

### PR DESCRIPTION
**Pull request by Prashanth684**
Follow up to #44025

### CODE REVIEW

File: core-services/release-controller/_releases/release-ocp-4.14-multi.json [hunk 1]
```diff
@@ -45,6 +45,13 @@
       "prowJob": {"name": "periodic-ci-openshift-multiarch-master-nightly-4.14-upgrade-from-stable-4.13-ocp-e2e-aws-ovn-heterogeneous-upgrade"},
       "upgrade": true,
       "upgradeFrom": "PreviousMinor"
+    },
+    "multi-azure-ovn-upgrade": {
+      "disabled": true,
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-ocp-4.14-multi.json:50]: The release-controller will completely ignore jobs that have `"disabled": true`.  Why bother even adding the job?

File: core-services/release-controller/_releases/release-ocp-4.15-multi.json [hunk 2]
```diff
@@ -49,6 +49,12 @@
       "prowJob": {
         "name": "periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-serial-aws-ovn-heterogeneous"
       }
+    },
+    "multi-azure-ovn-upgrade": {
+      "disabled": true,
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-ocp-4.15-multi.json:54]: The release-controller will completely ignore jobs that have "disabled": true. Why bother even adding the job?

File: core-services/release-controller/_releases/release-ocp-4.14-multi.json (hunk 1 above)
Comment (Prashanth684) [reply core-services/release-controller/_releases/release-ocp-4.14-multi.json]: oops..didn't mean to do that. lifted off the priv files

File: core-services/release-controller/_releases/release-ocp-4.15-multi.json (hunk 2 above)
Comment (Prashanth684) [reply core-services/release-controller/_releases/release-ocp-4.15-multi.json]: fixed

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**